Orange/widgets/data/owrandomize.py

```diff
@@ -0,0 +1,119 @@
+import random
+
+from AnyQt.QtCore import Qt
+from AnyQt.QtWidgets import QSizePolicy
+
+from Orange.data import Table
+from Orange.preprocess import Randomize
+from Orange.widgets.settings import Setting
+from Orange.widgets.widget import OWWidget
+from Orange.widgets import gui
+
+
+class OWRandomize(OWWidget):
+    name = "Randomize"
+    description = "Randomize features, class and/or metas in data table."
+    icon = "icons/Random.svg"
+    priority = 2100
+
+    inputs = [("Data", Table, "set_data")]
+    outputs = [("Data", Table)]
+
+    resizing_enabled = False
+    want_main_area = False
+
+    shuffle_class = Setting(1)
+    shuffle_attrs = Setting(0)
+    shuffle_metas = Setting(0)
+    scope_prop = Setting(80)
+    random_seed = Setting(0)
+    auto_apply = Setting(True)
+
+    def __init__(self):
+        super().__init__()
+        self.data = None
+
+        # GUI
+        box = gui.vBox(self.controlArea, "Shuffle")
```

Comment:
Vertical arrangement of check boxes with such short labels doesn't look nice. I think this is nicer:

        box = gui.hBox(self.controlArea, "Shuffled columns")
        box.layout().setSpacing(20)

That is, `hBox`, longer (more informative) title, and spacing.